src/background.js

```diff
@@ -1731,8 +1759,12 @@ function initializeGhosteryModules() {
 			// we need to do this after running scheduledTasks for the first time
 			// because of an A/B test that determines which promo variant is shown in the Hub on install
 			if (globals.JUST_INSTALLED) {
-				const route = (conf.hub_promo_variant === 'upgrade' || conf.hub_promo_variant === 'not_yet_set') ? '' : '#home';
-				const showPremiumPromoModal = conf.hub_promo_variant === 'midnight';
+				let route = (conf.hub_promo_variant === 'upgrade' || conf.hub_promo_variant === 'not_yet_set') ? '' : '#home';
+				let showPremiumPromoModal = conf.hub_promo_variant === 'midnight';
+				if (IS_ANDROID) {
```

Comment:
@wlycdgr Do you want me to add these to the ternary / declaration statements above?

Reply:
I think the `route` ternary would get too convoluted, but `const showPremiumPromoModal = (conf.hub_promo_variant === 'midnight' && !IS_ANDROID)` looks clear enough.